src/components/Navbar/Navbar.js

```diff
@@ -35,10 +67,10 @@ export default function Navbar(props) {
             <Link className="navLink" to="/AddItem">
               <AddItem />
             </Link>
-            <Link className="navLink" to="/">
+            <Link className="navLink" to="/" onClick={logoutHandler}>
```

Comment:
Akin to the previous comment maybe this becomes `{clearTokenHandler}` or something to make this flow clearer on the developer side as well.